firstvoices/backend/search/signals/dictionary_entry_signals.py

```diff
@@ -33,9 +34,16 @@ def remove_dictionary_entry_from_index(sender, instance, **kwargs):
 @receiver(post_save, sender=Acknowledgement)
 @receiver(
     post_save, sender=DictionaryEntryCategory
-)  # Category update when called through the admin site
+)  # Category update via creating m2m model (admin site does this)
```

Comment:
Since we now don't have a visible/editable admin form for dictionary entries, should we still keep these ? or is it safe to remove these ?